castervoice/rules/apps/editor/vscode_rules/vscode2.py

```diff
@@ -26,7 +26,7 @@ class VSCodeNonCcrRule(MappingRule):
         # Moving around a file
         "[(go to | jump | jump to)] line <n>":
             R(Key("c-g") + Text("%(n)d") + Key("enter")),
-        "<action> [line] <ln1> [by <ln2>]":
+        "<action> line <ln1> [by <ln2>]":
```

Comment:
line is not optional -> [line]

Reply:
Sorry about that. I thought I'd already removed it.